### ISSUE TITLE
Organized Catagories keep getting jumbled up

### ISSUE THREAD

**Issue by Peaches2003** [worldanvil/worldanvil-bug-tracker#1360]
**Please make sure to fill out the template to the best of your ability. Add your information to the right of the colon as requested by the label! More information helps us track down the issue and fix it. Bug reports that do not let us reproduce the issue cannot be addressed. Thank you for your time in helping make World Anvil better!**

**World Anvil Username**: Peaches2003

**World / Campaign / RPG Character / Manuscript Title**: Space Opera

**Feature**: Catagory

**Operating System**:

**Browser**:

**Device / Device Model**: 

**Related URLS** (To public & visible pages, add screenshots otherwise):

**Describe the Issue**
A clear and concise description of what the issue is.

So basically I have my categories organized in a specific way but it keeps getting made out of order when I haven't done the drag-and-drop thing so they keep being in the wrong order.

**To Reproduce**

**Expected behavior**

I would like m categories to not get jumbled out of order.

**Screenshots**
If applicable, add screenshots to help explain your problem. You can add screenshots by drag & drop into this window!
![Screenshot (52)](https://user-images.githubusercontent.com/104538785/202598124-72180785-4524-48af-9c93-38d1b68d1a4c.png)

**Comment by Kordishal** [comment]
Hi, this means that some of those categories have the same position / weight. When you click on each category you can adjust the position in the additional options to the right. Make sure each category has a unique value in there. They are ordered by highest number to lowest number. Rearranging them with drag and drop does not resolve it.
